apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -94,6 +94,23 @@ export default class TeacherHomepage extends Component {
         {hocLaunch &&
          hocLaunch.special_announcement &&
          hocLaunch.special_announcement === "celebs2017" &&
+         isEnglish && (
+          <SpecialAnnouncementActionBlock
+            isRtl={isRtl}
+            imageUrl={pegasus('/images/fill-540x289/special-announcements/celebs_hoc2017.jpg')}
+            heading={i18n.specialAnnouncementHeading()}
+            subHeading={""}
+            description={i18n.specialAnnouncementDescriptionCelebs()}
+            buttons={[
+              {url: pegasus('/challenge'), text: i18n.celebrityChallenge()},
+              {url: pegasus('/learn'), text: i18n.tryHOC()}
+            ]}
+          />
+        )}
+
+        {hocLaunch &&
+         hocLaunch.special_announcement &&
+         hocLaunch.special_announcement === "celebs2017actualhoc" &&
```

Comment:
Should this be a separate PR?  It seems like we should hotfix the "Hour of Code is here" change below, then push the announcement update as part of the normal DTP.

Reply:
Turns out that neither needs a hotfix, so we can just send them both out together as part of normal DTP.